content/desktop/contributing-and-collaborating-using-github-desktop/managing-branches.md

```diff
@@ -68,7 +68,6 @@ If you create a branch on {% data variables.product.product_name %}, you'll need
 
 ### Switching between branches
 You can view and make commits to any of your repository's branches. If you have uncommitted, saved changes, you'll need to decide what to do with your changes before you can switch branches. You can commit your changes on the current branch, stash your changes on the current branch, or bring the changes to your new branch. If you want to commit your changes on the current branch, follow the steps in "[Committing and reviewing changes to your project](/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project)" before switching branches.
```

Comment:
Since people can use the procedure in this article to stash changes and it seems closer to the context (someone is trying to switch branches using Desktop and this prompt appears), I think it makes more sense to keep the new "Stashing changes" article as further reading. Sending someone to another page at this point in the procedure would be a slower way to complete the task and possibly confusing.

Reply:
But I agree that some more context is needed on stashing so I'll explain that and have edited the link for localization 👍